changelogs/unreleased/4347-yankay-docs.md

```diff
@@ -0,0 +1 @@
+Update AWS NLB Guide with correct annotations. 
```

Comment:
Could you make this a bit more descriptive? The idea is that someone who reads this in the changelog will understand what the change has done.

Maybe something like:
```
The AWS deployment guide has been updated with the current Service annotations, and should now work correctly with the given YAMLs.
```

Reply:
Hi @youngnick , thanks for your comments, the changelog has been fixed.